nav2_simple_navigator/src/simple_navigator.cpp

```diff
@@ -26,26 +25,89 @@ namespace nav2_simple_navigator
 {
 
 SimpleNavigator::SimpleNavigator()
-: Node("SimpleNavigator")
+: nav2_lifecycle::LifecycleNode("simple_navigator")
+{
+  RCLCPP_INFO(get_logger(), "Creating");
+}
+
+SimpleNavigator::~SimpleNavigator()
 {
-  RCLCPP_INFO(get_logger(), "Initializing");
+  RCLCPP_INFO(get_logger(), "Destroying");
+}
 
-  auto temp_node = std::shared_ptr<rclcpp::Node>(this, [](auto) {});
+nav2_lifecycle::CallbackReturn
+SimpleNavigator::on_configure(const rclcpp_lifecycle::State & state)
+{
+  RCLCPP_INFO(get_logger(), "on_configure");
```

Comment:
Can we make these messages "Configuring", "Activating", etc?

Reply:
Sure.